src/snapred/backend/recipe/ApplyNormalizationRecipe.py

```diff
@@ -23,8 +23,11 @@ def chopIngredients(self, ingredients: Ingredients):
         We are mostly concerned about the drange for a ResampleX operation.
         """
         self.pixelGroup = ingredients.pixelGroup
-        self.dMin = self.pixelGroup.dMin()
-        self.dMax = self.pixelGroup.dMax()
+        # The adjustment below is a temp fix, will be permanently fixed in EWM 6262
+        dMin = [x + Config["constants.CropFactors.lowdSpacingCrop"] for x in self.pixelGroup.dMin()]
+        dMax = [x - Config["constants.CropFactors.highdSpacingCrop"] for x in self.pixelGroup.dMax()]
+        self.dMin = dMin
```

Comment:
We need a test here to make sure that `dMax > dMin` after this change.  If it's not I'd then
`raise ValueError("d-spacing crop factors are too large -- resultant dMax must be > resultant dMin")`